tools/libboard/board_n6010/board_n6010.h

```diff
@@ -34,6 +34,13 @@ extern "C" {
 #endif /* __cplusplus */
 
 #define SYSFS_MAX_SIZE             256
+#define MAC_BYTE_SIZE              4
+#define MAC_BUF_LEN                19
+
+typedef union _cvl_mac {
+	unsigned int dword;
+	unsigned char byte[MAC_BYTE_SIZE];
```

Comment:
ethernet addresses are typically defined as 6 byte quantities.  Why not do the normal approach?

Reply:
fixed ,removed struct cvl_mac